setup.py

```diff
@@ -7,113 +7,114 @@
 from setuptools import find_packages, setup
 
 install_requires = [
-    'coloredlogs',
```

Comment:
I added a question of whether we want to stick with single vs double quote. I'd love to pick one and move towards it.

Reply:
Agreed.